src/clarity-angular/datagrid/providers/selection.ts

```diff
@@ -53,17 +103,21 @@ export class Selection {
     /**
      * Ignore items changes in the same change detection cycle.
      */
-    private debounce = false;
+    private debounce: boolean = false;
```

Comment:
We're not using this anymore, are we? If that's the case, we want to remove it altogether.